rust/src/client.rs

```diff
@@ -91,6 +111,117 @@ struct SessionManager {
     tx: tokio::sync::mpsc::Sender<command::Command>,
 }
 
+impl SessionManager {
+    async fn send(
+        &self,
+        message: &models::MessageImpl,
+        client: Weak<&Client>,
+    ) -> Result<SendMessageResponse, ClientError> {
+        let client = match client.upgrade() {
+            Some(client) => client,
+            None => {
+                return Err(error::ClientError::ClientInternal);
+            }
+        };
+
+        let mut srequest = SendMessageRequest {
+            messages: vec![],
+        };
+
+
+        let mut delivery_timestamp = None;
+
+        if message.deliveryTimestamp != 0 {
+            delivery_timestamp = Option::from(Timestamp {
+                seconds: message.deliveryTimestamp,
+                nanos: 0,
+            })
+        }
+
+        srequest.messages.push(pb::Message {
+            topic: Some(Resource {
+                resource_namespace: "".to_string(),
+                name: message.topic.clone(),
+            }),
+            user_properties: Default::default(),
+            system_properties: Option::from(pb::SystemProperties {
+                tag: Option::from(message.tags.clone()),
+                keys: message.keys.clone(),
+                message_id: "123".to_string(),
```

Comment:
It would be great if the message id could be generated according to the rules [here](https://github.com/apache/rocketmq-clients/blob/master/docs/message_id.md).

Reply:
> It would be great if the message id could be generated according to the rules [here](https://github.com/apache/rocketmq-clients/blob/master/docs/message_id.md).

done，someone can run the test case to verify it